app/lib/event_store/active_record_event_repository.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module EventStore
+  class ActiveRecordEventRepository < RailsEventStoreActiveRecord::EventRepository
+    def build_event_entity(record)
+      return nil unless record
+
+      data = (record.data || {}).merge(
```

Comment:
maybe add a comment that the only actual change from the overridden method is handling `nil` `record.data` gracefully.

Reply:
Actually, that was not the purpose :upside_down_face: 

The actual reason the method is overriden is converting the hash to kwargs on line `:13` (and there is a comment on line `:12`).
However, I think in some tests I saw this `record.data` being nil, so I added this small tweak also. But it is weird in fact... because it used to work, so it shouldn't be needed. I may take another look on whether it's needed or not.